manifest.json

```diff
@@ -10,20 +13,49 @@
     "name": "Lapineige"
   },
   "url": "https://github.com/shaarli/Shaarli",
+  "license": "free",
   "version": "0.8.3",
-  "multi_instance": "false",
+  "multi_instance": "true",
   "arguments": {
     "install" : [
     {
       "name": "domain",
+      "type": "domain",
       "ask": {
         "en": "Choose a domain for your Shaarli",
         "fr": "Choisissez un domaine pour votre Shaarli"
       },
       "example": "domain.org"
     },
     {
+      "name": "is_public",
+      "ask": {
+        "en": "Is it a public Shaarli site ?",
+        "fr": "Est-ce un site Shaarli public ?"
+      },
+      "choices": ["Yes", "No"],
+      "default": "Yes"
+    },
+    {
+      "name": "admin",
+      "type": "user",
+      "ask": {
+        "en": "Owner of the Shaarli instance",
+        "fr": "Propriétaire de l'instance Shaarli"
+      },
+      "example": "test"
+    },
+    {
+      "name": "password",
+      "type": "password",
+      "ask": {
+        "en": "Define password for Shaarli user",
```

Comment:
Ok giving a blank password is not possible. So we can just tell the user to use a random password as it is not required.